Update upgrade known issue about apply-manifest stuck (#423)
harvester/docs: docs/upgrade/v1-1-2-to-v1-2-0.md

    Tue Sep  5 10:20:39 UTC 2023
    there are still 1 pods in cattle-monitoring-system to be deleted
    Tue Sep  5 10:20:45 UTC 2023
    there are still 1 pods in cattle-monitoring-system to be deleted
    Tue Sep  5 10:20:50 UTC 2023
    there are still 1 pods in cattle-monitoring-system to be deleted
    Tue Sep  5 10:20:55 UTC 2023
    there are still 1 pods in cattle-monitoring-system to be deleted
    Tue Sep  5 10:21:00 UTC 2023
    there are still 1 pods in cattle-monitoring-system to be deleted
    ```

1. Check if the `prometheus-rancher-monitoring-prometheus-0` Pod is stuck in the `Terminating` status

    ```
    $ kubectl -n cattle-monitoring-system get pods                                  
    NAME                                         READY   STATUS        RESTARTS   AGE
    prometheus-rancher-monitoring-prometheus-0   0/3     Terminating   0          19d
    ```

1. Find out the terminating Pod's UID

    ```
    $ kubectl -n cattle-monitoring-system get pod prometheus-rancher-monitoring-prometheus-0 -o jsonpath='{.metadata.uid}'
    33f43165-6faa-4648-927d-69097901471c
    ```

1. Get access to any node of the cluster via console or SSH

> 1. Search for the related log messages in `/var/lib/rancher/rke2/agent/logs/kubelet.log` using the Pod's UID

Reviewer: ```suggestion
1. Search for the related log messages in `/var/lib/rancher/rke2/agent/logs/kubelet.log` using the pod's UID.
```

    ```

>     If kubelet keeps complaining about the volume unmount failure, you might want to apply the following workaround to make the upgrade proceed.

Reviewer: ```suggestion
     If kubelet continues to complain about the volume failing to unmount, apply the following workaround to allow the upgrade to proceed.
```
